contrib/pkg/createcluster/create.go

```diff
@@ -392,6 +402,42 @@ func (o *Options) Validate(cmd *cobra.Command) error {
 			return fmt.Errorf("unable to parse key=value label: %s", ls)
 		}
 	}
+
+	if len(o.HttpProxy) > 0 || len(o.HttpsProxy) > 0 {
+		if o.HttpProxy == "" && o.HttpsProxy == "" {
+			cmd.Usage()
+			log.Info("must set --http-proxy or --https-proxy")
+			return fmt.Errorf("must set --http-proxy or --https-proxy")			
+		}
+	}
+		
+	if len(o.HttpProxy) > 0 && o.HttpProxy != "" {
+		if err := validate.URI(o.HttpProxy); err != nil {
+			cmd.Usage()
+			log.Info("--http-proxy specified in wrong format")
+			return fmt.Errorf("--http-proxy specified in wrong format: %s", err.Error())
+		}
+	}
+	if len(o.HttpsProxy) > 0 && o.HttpsProxy != "" {
+		if err := validate.URI(o.HttpsProxy); err != nil {
+			cmd.Usage()
+			log.Info("--https-proxy specified in wrong format")
+			return fmt.Errorf("--https-proxy specified in wrong format: %s", err.Error())
+		}
+	}
+	if len(o.NoProxy) > 0 && o.NoProxy != "" {
+		for _, v := range strings.Split(o.NoProxy, ",") {
+			v = strings.TrimSpace(v)
+			errDomain := validate.NoProxyDomainName(v)
+			_, _, errCIDR := net.ParseCIDR(v)
+			if errDomain != nil && errCIDR != nil {
+				cmd.Usage()
+				log.Info("--no-proxy specified in wrong format - must be a CIDR or domain, without wildcard characters")
```

Comment:
Looks like this should be "must be a comma separated list of CIDR domains without wildcard characters"